### DISCUSSION TITLE
Multiple Cameras

### DISCUSSION THREAD

**Discussion by ztef** [vsg-dev/VulkanSceneGraph#860] [General]
I've been playing with vsgcameras example.

Seems to create a main camera and left, rigth, front, top ones ok.

but once the main loop starts, even when the CameraSelector handle the keyboard keys, never switch to those cameras.

**Comment by robertosfield** [comment]
What do you mean by "never switch to those cameras"?  

Can you see the additional cameras OK?

What are you expecting to happen?

**Comment by ztef** [comment]
I am expecting to see the scene trough the new camera parameters, i mean see the scene from the left, the top etc, but never happens

**Comment by vsg-dev** [comment]
The vsgcamera example only associates a single vsg::Trackball with the main vsg::Camera, there are no additional Trackball's assigned to the other views so they are static:

https://github.com/vsg-dev/vsgExamples/blob/master/examples/app/vsgcameras/vsgcameras.cpp#L199

The vsgmultiviews example creates a main and insert view each with their own vsg::Trackball so have independent control:

https://github.com/vsg-dev/vsgExamples/blob/master/examples/app/vsgmultiviews/vsgmultiviews.cpp#L150
